components/combobox/private/menu.jsx

```diff
@@ -20,9 +20,13 @@ const propTypes = {
 	 */
 	activeOptionIndex: PropTypes.number,
 	/**
-	 * CSS classes to be added to tag with `.slds-dropdown`. Uses `classNames` [API](https://github.com/JedWatson/classnames).
+	 * CSS classes to be added to container `div` tag. Uses `classNames` [API](https://github.com/JedWatson/classnames).
 	 */
 	className: PropTypes.oneOfType([PropTypes.array, PropTypes.object, PropTypes.string]),
+	/**
+	 * CSS classes to be added to tag with `.slds-dropdown`. Uses `classNames` [API](https://github.com/JedWatson/classnames).
+	 */
+	classNameMenu: PropTypes.oneOfType([PropTypes.array, PropTypes.object, PropTypes.string]),
```

Comment:
I don't love that we're going against the pattern of the parent container getting`containerClassName` and its child getting `className` but I understand that you don't want to break things. I think I'd prefer having a standard pattern across all components but what are your thoughts?

Reply:
This is private. It can break. This may have just been an oversight. I'll update.